lib/dl_core/dl_core/us_manager/schema_migration/base.py

```diff
@@ -1,17 +1,23 @@
+from __future__ import annotations
```

Comment:
let's avoid using __future__.annotations
it's a kludge for back compatability

why do we need it here?